.github/workflows/cifuzz.yml

```diff
@@ -3,6 +3,7 @@ on:
   pull_request:
     paths:
       - 'include/**'
+      - 'ext/**'
```

Comment:
Nit: List "ext" before "include"

Reply:
Fixed in https://github.com/AOMediaCodec/libavif/pull/2092